BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/DigitalInvoiceCoordinator.swift

```diff
@@ -23,7 +23,7 @@ final class DigitalInvoiceCoordinator: Coordinator {
 
     // TODO: This is to cope with the screen coordinator being inadequate at this point to support the return assistant step and needing a refactor.
     // Remove ASAP
-    private var analysisDelegate: AnalysisDelegate
+    weak private var analysisDelegate: AnalysisDelegate?
```

Comment:
[suggestions]: Can we order like this: `private weak var` ? 
Starting with the access control modifier is generally considered a good practice for readability and consistency.

Reply:
I've used the automatic function...looks like it's not following the convention :)